e2e_test/src/independent/restart_tests.rs

```diff
@@ -764,3 +775,115 @@ fn case6() {
         },
     ]);
 }
+
+#[test]
+fn case8_count_after_restart_cluster() {
+    println!("Test begin case8 count after restart cluster");
+
+    let test_dir = PathBuf::from("/tmp/e2e_test/independent/restart/case8");
+    let _ = std::fs::remove_dir_all(&test_dir);
+    std::fs::create_dir_all(&test_dir).unwrap();
+
+    kill_all();
+
+    fn start_cluster(
+        test_dir: &PathBuf,
+        runtime: Arc<tokio::runtime::Runtime>,
+    ) -> (Option<CnosdbMetaTestHelper>, Option<CnosdbDataTestHelper>) {
+        run_cluster_with_customized_configs(
+            test_dir,
+            runtime,
+            &CnosdbClusterDefinition::with_ids(&[1], &[1, 2]),
+            true,
+            true,
+            vec![Some(Box::new(|c| {
```

Comment:
This path of meta node is the parent path of expected path:

```
meta
├── 1_data
│   ├── data.mdb
│   └── lock.mdb
├── 1_entry
│   ├── data.mdb
│   └── lock.mdb
├── 1_state
│   ├── data.mdb
│   └── lock.mdb
├── config
│   └── config_8901.toml
└── config_8901.toml
    └── log
        └── 1
            └── meta_server.log.2024-07-11
```

Why not use the default path?

```
meta
├── config
│   └── config_8901.toml
└── config_8901.toml
    ├── log
    │   └── 1
    │       └── meta_server.log.2024-07-11
    └── meta
        ├── 1_data
        │   ├── data.mdb
        │   └── lock.mdb
        ├── 1_entry
        │   ├── data.mdb
        │   └── lock.mdb
        └── 1_state
            ├── data.mdb
            └── lock.mdb
```